tensorflow/inference/docker/1.15.3/py3/Dockerfile.cpu

```diff
@@ -4,6 +4,7 @@ LABEL maintainer="Amazon AI"
 # Specify LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
 # https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
Do we need the similar line in GPU dockerfiles as well ?

Reply:
MME is not supported to GPU yet.